### DISCUSSION TITLE
Can you change formatting for inline-markdown links?

### DISCUSSION THREAD

**Discussion by scapeg0at** [artempyanykh/marksman#240] [Q&A]
Hello everyone,
I've started using Marksman to build up a personal wiki and now I'm trying to render it into a static webpage using Hugo.
The issue I'm running into is that Hugo doesn't like the style of inline links Marksman autocompletes, namely that the URI for the link has a file extension.
I know that it is possible to change how wiki style links are auto-completed: title-slug, file-name, and file path. I'm wondering if there is there a similar setting I just cannot seem to find that can change the auto-complete for in-line marksman links.

Here's an example of what I would like:
```markdown
Current link style
[link name](/path/to/markdown/file.md)

What I would like
[link name](/path/to/markdown/file)
```

Or, if it would be easier, does anyone know what settings to change in Hugo for it to parse the inline links properly if they have the file extension?

Here's some info on my setup:
`marksman version: 1.0.0-97da2ea-dirty`
`Neovim version: v0.9.1`

**Comment by artempyanykh** [comment]
Hi @scapeg0at! Thanks for asking!

Currently, there's no config option to skip the extension for inline links. It's a reasonable ask though, so feel free to open an issue about this.

When it comes to Hugo, this is somewhat relevant I think: https://discourse.gohugo.io/t/how-can-i-convert-markdown-files-contains-relative-link-with-end-with-md-to-hugo-links/42087/2 But by the looks of it it seems that Hugo wants cross-file links to be of [its own format](https://gohugo.io/content-management/cross-references/).
